### PR TITLE
Skeleton code for State class

### PR DESCRIPTION
**Pull request by amitrahman1026**

### CODE REVIEW

File: src/main/java/seedu/duke/utils/State.java [hunk 1]
```diff
@@ -1,5 +1,19 @@
 package seedu.duke.utils;
 
+import seedu.duke.model.SelectedModule;
+
+import java.util.List;
+
 public class State {
 
+    public int semester = 0;
```
Comment (owenl131) [inline src/main/java/seedu/duke/utils/State.java:9]: I think this should be private with getter and setter

Comment (amitrahman1026) [reply src/main/java/seedu/duke/utils/State.java]: updated, thanks

**Review by owenl131 — APPROVED** [review]
LGTM

**Comment by amitrahman1026** [comment]
Grape

**Pull request merged**